style.css

```diff
@@ -796,4 +781,133 @@ input[type='submit']:active {
   * {
     margin: auto;
   }
+
+  .menu {
+    display: none;
+  }
+
+  .desktop-menu {
+    display: flex;
+    align-items: center;
+    gap: 30px;
+  }
+
+  .desktop-menu li a {
+    color: #344563;
+    text-align: center;
+    font-family: Poppins, sans-serif;
+    font-size: 15px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 20px;
+    text-decoration: none !important;
+  }
+
+  .headline-container {
+    background-image: url(assets/images/desktop-header-bg.png);
+  }
+
+  .headline {
+    margin-right: 20%;
+    margin-left: 20%;
+  }
+
+  .social-media-icons {
+    display: inline-flex;
+  }
+
+  .works {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    width: 85%;
+    flex: 1;
+  }
+
+  .work-info {
+    display: flex;
+    align-items: center;
+    padding: 0;
+    gap: 12px;
+    width: auto;
+    margin-left: initial;
+  }
+
+  .language-tags {
+    margin-left: initial;
+  }
+
+  .about-container {
+    width: 100%;
+  }
+
+  .about-myself {
+    padding: 114px 120px;
+    flex-direction: row;
+  }
+
+  .skill-list {
+    width: 100%;
+  }
+
+  .dropdown-content {
+    flex-direction: row;
+  }
+
+  .dropdown-content li {
+    display: block;
+    width: 100px;
+    margin-left: 10px;
+    margin-right: 10px;
+  }
+
+  .top {
+    width: 100%;
+    max-width: 50%;
+    margin-left: 0;
+    margin-right: 0;
+  }
+
+  .top h2,
+  p,
```

Comment:
-- Hey, let's improve the font sizes of the elements in the desktop version as yours is still not close. below is an example of the paragraph in the headline section. please, improve on all the font sizes to make your design better:


![image](https://github.com/NoelLincoln/myportfolio/assets/59397159/61da0900-cfed-4b11-a933-d71e99751b33)